test/fixtures/rules/std_rule_cases/L059.yml

```
@@ -47,3 +80,189 @@ test_pass_special_symbols:
 test_pass_reserved_keyword:
   pass_str: |
     SELECT 123 AS "SELECT"
+
+test_pass_column_reference_forced_ansi:
+  pass_str: |
+    SELECT 123 AS "foo";
+  configs:
+    rules:
+      L059:
+        force_quote_identifier: True
```

Can we change all the other test cases like this?

```suggestion
test_pass_column_reference_prefer_quoted_ansi:
  pass_str: |
    SELECT 123 AS "foo";
  configs:
    rules:
      L059:
        prefer_quoted_identifiers: True
```

---

Sure, will do!

---

Done in: https://github.com/niconoe-/sqlfluff/commit/a5e60fce2af064733ca36c2b5d68444714c37971